app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -60,6 +89,7 @@ def perform_vmware_to_ovirt_import(params)
           :sparse         => params[:sparse],
         )
       )
+      make_ems_ref(import.vm.href)
```

Comment:
yes the tests are indeed failing, but it works when deployed and run within the EVM.... could it be possible that the tests just don't see methods defined in the `api_integration` mixin? @borod108 @jhernand